modulefiles/ufs_gaea.intel.lua

```diff
@@ -0,0 +1,33 @@
+help([[
+  This module loads libraries required for building and running UFS Weather Model
+  on the NOAA RDHPC machine Gaea C5 using Intel-2023.1.0.
+]])
+
+whatis([===[Loads libraries needed for building the UFS Weather Model on Gaea ]===])
+
+prepend_path("MODULEPATH", "/ncrc/proj/epic/spack-stack/spack-stack-1.6.0/envs/fms-2024.01/install/modulefiles/Core")
+
+stack_intel_ver=os.getenv("stack_intel_ver") or "2023.1.0"
```

Comment:
Tried C5 and C6 and it seems like only 2023.2.0 is available.
```suggestion
stack_intel_ver=os.getenv("stack_intel_ver") or "2023.2.0"
```

Reply:
https://github.com/ufs-community/ufs-weather-model/commit/db1781a05dce1125cfe17f8324650674640f0a9e